contrib/git-sync/demo/config/pod.yaml

```diff
@@ -1,50 +1,50 @@
-id: blog-pod
+apiVersion: v1beta3
 kind: Pod
-apiVersion: v1beta1
-desiredState:
-  manifest:
-    version: v1beta1
-    containers:
-    - name: git-sync
-      image: proppy/git-sync
-      imagePullPolicy: PullAlways
-      env:
-      - name: GIT_SYNC_REPO
-        value: https://github.com/GoogleCloudPlatform/kubernetes.git
-      - name: GIT_SYNC_DEST
-        value: /git
-      volumeMounts:
-      - name: markdown
-        mountPath: /git
-    - name: hugo
-      image: proppy/hugo
-      imagePullPolicy: PullAlways
-      env:
-      - name: HUGO_SRC
-        value: /src/contrib/git-sync/demo/blog
-      - name: HUGO_BUILD_DRAFT
-        value: 'true'
-      - name: HUGO_BASE_URL
-        value: example.com
-      volumeMounts:
-      - name: markdown
-        mountPath: /src
-      - name: html
-        mountPath: /dest
-    - name: nginx
-      image: nginx
-      volumeMounts:
-      - name: html
-        mountPath: /usr/share/nginx/html
-      ports:
-      - name: http-server
-        containerPort: 80
-    volumes:
-    - name: markdown
-      source:
-        emptyDir: {}
-    - name: html
-      source:
-        emptyDir: {}
-labels:
-  name: blog
+metadata:
+  labels:
+    name: blog
+  name: blog-pod
+spec:
+  containers:
+  - env:
+    - name: GIT_SYNC_REPO
+      value: https://github.com/GoogleCloudPlatform/kubernetes.git
+    - name: GIT_SYNC_DEST
+      value: /git
+    image: gcr.io/google_containers/git-sync
+    imagePullPolicy: Always
+    name: git-sync
+    volumeMounts:
+    - mountPath: /git
+      name: markdown
+  - env:
+    - name: HUGO_SRC
+      value: /src/contrib/git-sync/demo/blog
+    - name: HUGO_BUILD_DRAFT
+      value: "true"
+    - name: HUGO_BASE_URL
+      value: example.com
+    image: proppy/hugo
+    imagePullPolicy: Always
+    name: hugo
+    volumeMounts:
+    - mountPath: /src
+      name: markdown
+    - mountPath: /dest
+      name: html
+  - image: nginx
+    name: nginx
+    ports:
+    - containerPort: 80
+      name: http-server
+      protocol: TCP
+    volumeMounts:
+    - mountPath: /usr/share/nginx/html
+      name: html
+  volumes:
+  - emptyDir:
```

Comment:
Please write emptyDir the way it was written above, with `{}`, and eliminate `medium`, since that's just the default value.